Fix: spacing between cards

swagataroy30 opened the pull request:
**Description**

This PR fixes #3759 

**Notes for Reviewers**

## Before 
![Screenshot from 2023-02-20 01-19-05](https://user-images.githubusercontent.com/116095538/219972669-1ca730a2-7aaf-4576-99c8-3827a6309bcb.png)

## After
![Screenshot from 2023-02-20 01-16-55](https://user-images.githubusercontent.com/116095538/219972725-10800c26-6f4b-4603-839b-1fb5e8a6e85a.png)


**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Layer5 projects! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

l5io commented:
🚀 Preview for commit 37462591b17bc12c08c8173e140031466c8f7cda at: https://63f2866e464f7c2c77e9f866--layer5.netlify.app

leecalcote commented:
Tagging @avikt18 and @avi-88 for review.

Avi-88 reviewed (changes requested):
It looks good. This does solve the issue at hand but  , I was just curious about the implementation of the cards as it would've been much better if we used css grid instead of just using flexbox with random margins and paddings to make it fit. Let me know what you think about that. Also while you are at it could you please change the border radius of the card to be a value instead of percentage. It's looking wierd for a single column layout for certain screen sizes like so

![image](https://user-images.githubusercontent.com/79744258/220729524-9483d072-51fb-41b3-b0e0-11195aa196ce.png)

I guess 10px should roughly provide the same effect instead of 4% , thanks

swagataroy30 commented:
> It looks good. This does solve the issue at hand but , I was just curious about the implementation of the cards as it would've been much better if we used css grid instead of just using flexbox with random margins and paddings to make it fit. Let me know what you think about that. Also while you are at it could you please change the border radius of the card to be a value instead of percentage. It's looking wierd for a single column layout for certain screen sizes like so
> 
> ![image](https://user-images.githubusercontent.com/79744258/220729524-9483d072-51fb-41b3-b0e0-11195aa196ce.png)
> 
> I guess 10px should roughly provide the same effect instead of 4% , thanks

Hey sorry for the late reply ... Thanks @Avi-88 for the suggestions will change the border radius as said.

Chadha93 commented:
@swagataroy30  Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

avikt18 reviewed (commented):
@swagataroy30 I guess making changes to a reusable component (`src/reusecore/Layout/Col.js` in this case) is not the best thing you can do. You can make changes to the projectSection.style.js instead. 
@Chadha93 pls correct me if I am wrong :sweat_smile:

swagataroy30 commented:
Removed changes that was done to the reusable component and moved changes to projectSection.style.js ... thanks @avikt18 for the suggestion. Fixed space between cards and also the border radius.

l5io commented:
🚀 Preview for commit 707276a1ca3eed67ce01e58ea8c1c1ddca3e93c5 at: https://63fd030ca32b760a15fb9fc3--layer5.netlify.app

swagataroy30 commented:
![Screenshot from 2023-02-28 00-41-16](https://user-images.githubusercontent.com/116095538/221659980-95841021-bfb5-4f08-a915-c16e9757cdae.png)

Hey as you can see the below media query is a duplicate for 992px and also the css mentioned in it seems to be unused ... should i remove it ?

Nikhil-Ladha commented:
> ![Screenshot from 2023-02-28 00-41-16](https://user-images.githubusercontent.com/116095538/221659980-95841021-bfb5-4f08-a915-c16e9757cdae.png)
> 
> Hey as you can see the below media query is a duplicate for 992px and also the css mentioned in it seems to be unused ... should i remove it ?

Please merge the duplicate media queries into one, and if you think the CSS is unused feel free to remove it and push the changes.

leecalcote reviewed (changes requested):
Pending incorporation of Nikhil's feedback.

swagataroy30 commented:
> Pending incorporation of Nikhil's feedback.

Sorry for the late reply was busy with my exams. Made the said changes!

l5io commented:
🚀 Preview for commit 1b3ac07921c83ba719b9a6c94e3dd66664c2fa73 at: https://6415fb0bdf97a603c92dfa49--layer5.netlify.app

Udit-takkar reviewed (commented):
@swagataroy30 @Nikhil-Ladha has this been already fixed in some other PR?

This is what i see on https://layer5.io/

<img width="801" alt="Screenshot 2023-04-10 at 2 59 47 AM" src="https://user-images.githubusercontent.com/53316345/230797367-473501f4-1456-45bc-9a96-06861ae56a96.png">

swagataroy30 commented:
But the issue still persists as i see..
![Screenshot from 2023-04-10 08-07-37](https://user-images.githubusercontent.com/116095538/230814501-206a3ca3-ef61-41bc-b44f-aeee304465de.png)

Nikhil-Ladha commented on src/sections/Home/Projects-home/projectSection.style.js:
```diff
@@ -111,11 +106,8 @@ const ProjectItemWrapper = styled.section`
-            font-size: 7rem;
-            color:red;
+        .project__block__inner {
+            margin-bottom: 40px;
```
Make it `2.5rem` instead

swagataroy30 replied:
Done!

l5io commented:
🚀 Preview for commit ffd36fd9e1879785edbfd523044c9025589dadcf at: https://6434cec4db97f7611014848f--layer5.netlify.app
